package.json

```diff
@@ -45,5 +45,13 @@
     "router",
     "route-recognizer",
     "rsvp"
-  ]
+  ],
+  "dependencies": {
```

Comment:
These are required to run `router.js` from a frontend web application, so it seems like they actually are dependencies and not dev dependencies?